src/api/profileApi.jsx

```diff
@@ -0,0 +1,27 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
+export const profileApi = createApi({
+  reducerPath: "profileApi",
+  baseQuery: fetchBaseQuery({
+    baseUrl: process.env.REACT_APP_API_URL,
+    prepareHeaders: (headers, { getState }) => {
+      const token = getState().auth.token;
+      if (token) {
+        headers.set("authorization", `Bearer ${token}`);
+      }
+      return headers;
+    },
+  }),
+  tagTypes: ["profile"],
+  endpoints: (builder) => ({
+    getProfileList: builder.query({
+      query: () => ({
+        url: `/api/list_profiles`,
```

Comment:
move to constant and use
do not use use less string interpolation quotes.

Reply:
done